### PR TITLE
editorial: Replace 'package repo' with 'package registry', following the recommended terminology.

### PR DESCRIPTION
**Pull request by eruvanos**
Replace 'package repo' with 'package registry', following the recommended terminology.

### CODE REVIEW

File: docs/images/supply-chain-threats-build-verification.svg
Comment (MarkLodato) [inline docs/images/supply-chain-threats-build-verification.svg:1]: Oops, this file is only used by v0.1 and v1.0-rc*. Could you please revert this file and update `docs/spec/v1.[12]/images/supply-chain-threats-build-verification.svg` instead? Thanks.

File: docs/_includes/use-cases-table.html
Comment (MarkLodato) [inline docs/_includes/use-cases-table.html:1]: Oh, I didn't realize this file still existed. It's unused. I just sent #984 to delete it. You can revert this file.

File: docs/images/supply-chain-threats.svg
Comment (MarkLodato) [inline docs/images/supply-chain-threats.svg:1]: same

File: docs/spec/v1.1/threats-overview.md
Comment (MarkLodato) [inline docs/spec/v1.1/threats-overview.md:1]: Do you mind updating v1.0 as well for the these three files?

**Review by MarkLodato — CHANGES_REQUESTED** [review]
This is great, thank you so much for your contribution! There are some slight tweaks to make, but otherwise this change looks good.

File: docs/spec/v1.1/threats-overview.md
Comment (eruvanos) [reply docs/spec/v1.1/threats-overview.md:1]: Done

**Comment by eruvanos** [comment]
Applied the requested changes, please double check the right images are replaced now.

**Review by MarkLodato — APPROVED** [review]
Perfect, thank you so much!

**Review by joshuagl — APPROVED** [review]
This is great, thank you!

**Comment by eruvanos** [comment]
@joshuagl anything missing from my side?

**Comment by joshuagl** [comment]
Nothing missing, thanks again for the contribution @eruvanos

**Pull request merged**